maxim/logger/portkey/client.py

```
@@ -0,0 +1,38 @@
+from typing import Union
+
+try:
+    from portkey_ai.api_resources.client import AsyncPortkey, Portkey
+except ImportError as e:
+    raise ImportError(
+        (
+            "The 'portkey-ai' package is required for Portkey integration. "
+            "Install it with `pip install portkey-ai` or `uv add portkey-ai`."
+        )
+    ) from e
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)